### PR TITLE
Start API documentation using Swagger

### PR DESCRIPTION
**Pull request by GreatWizard**
need swagger-UI to render the file

How to use: http://editor.swagger.io/#!/

### CODE REVIEW

File: docs/swagger/formidable.yml [hunk 1]
```diff
@@ -0,0 +1,478 @@
+# this is an example of the Uber API
+# as a demonstration of an API spec in YAML
```
Comment (wo0dyn) [inline docs/swagger/formidable.yml]: We should customize this comment. :wink:

**Comment by wo0dyn** [comment]
I have embed Swagger UI into our sphinx documentation. How does this look for you? · ping @mgu @GreatWizard 

screencapture-file-home-wo0dyn-projects-djangoformidable-docs-build-html-api-html-1488964121830

**Comment by moumoutte** [comment]
:+1: ! But I don't understand what the /clien/fake_path/form/{id}/ is !

If the objectif is to match with Peopleask guideline , I'm completely against put the URL on django-formidable.

**Comment by mgu** [comment]
The objectif is to reflect the API between formidable-ui and django-formidable.
But formidable-ui can be configured to load an URL that is not part of django-formidable (to retrieve a form with values).
How would you document this behavior ?

**Comment by moumoutte** [comment]
To document it in another part. django-formidable does not have to embed peopleask specification. We can document a part in order to configure a custom URL part with View Inheritance provided by django-formidable.

File: docs/Makefile [hunk 2]
```diff
@@ -228,3 +229,21 @@ dummy:
 	$(SPHINXBUILD) -b dummy $(ALLSPHINXOPTS) $(BUILDDIR)/dummy
 	@echo
 	@echo "Build finished. Dummy builder generates no files."
+
+.PHONY: collectstatic
+collectstatic:
+	npm install
```
Comment (GreatWizard) [inline docs/Makefile]: Is npm avalaible on CircleCI ?

Comment (wo0dyn) [inline docs/Makefile]: screencapture-circleci-gh-novafloss-django-formidable-748-1488986984853

I tried to build the docs on CircleCI: worked pretty well!

**Pull request merged**